src/main/java/io/jenkins/plugins/analysis/core/columns/IssuesTotalColumn.java

```diff
@@ -264,12 +264,18 @@ public ListBoxModel doFillTypeItems() {
             this.url = url;
         }
 
-        AnalysisResultDescription(final ResultAction result, final LabelProviderFactory labelProviderFactory) {
+        AnalysisResultDescription(final ResultAction result, final LabelProviderFactory labelProviderFactory,
+                final StatisticProperties type) {
             StaticAnalysisLabelProvider labelProvider = labelProviderFactory.create(result.getId(), result.getName());
             name = labelProvider.getLinkName();
             icon = labelProvider.getSmallIconUrl();
             total = result.getResult().getTotalSize();
-            url = result.getUrlName();
+
+            String typeUrl = type.name().split("_")[0];
+            if (!typeUrl.matches("NEW|FIXED")) {
+                typeUrl = "";
+            }
+            url = result.getOwner().getNumber() + "/" + result.getUrlName() + "/" + typeUrl;
```

Comment:
I would expect that some tests fail after your change: https://github.com/jenkinsci/warnings-ng-plugin/blob/master/src/test/java/io/jenkins/plugins/analysis/core/columns/IssuesTotalColumnTest.java#L83

There should be a reference to `new`? Also it would be nice to have a test that uses a severity.

Reply:
It passes the tests because I forgot changing the `getUrl()` in `IssuesTotalColumn`from the beginning :smile: 

I'll change it this time.